tools/scripts/create-json-for-manifest.ps1

```diff
@@ -104,10 +108,11 @@ function Get-MinimumProductionVersion([string]$currentVersion, $isMandatoryProdU
     return $minimumProductionVersion
 }
 
-function CreateJsonForManifest([string]$msiBasePath, [string]$isMandatoryProdUpdate) {
+function CreateJsonForManifest([string]$msiBasePath, [string]$octokitPath, [string]$isMandatoryProdUpdate) {
     Write-Verbose "Entering CreateJsonForManifest"
     Write-Verbose "  msiBasePath = $msiBasePath"
-    Write-Verbose "  $isMandatoryProdUpdate = $isMandatoryProdUpdate"
+    Write-Verbose "  octokitPath = $octokitPath"
+    Write-Verbose "  isMandatoryProdUpdate = $isMandatoryProdUpdate"
```

Comment:
Removing the `$` here fixes a typo. It was supposed to output 
```
  isMandatoryProdUpdate = false
```
but was outputting
```
  false = false
```